packages/terra-application/src/application-intl/ApplicationIntlContext.jsx

```diff
@@ -1,9 +1,5 @@
 import { createContext } from 'react';
-import { intlShape } from 'react-intl';
 
 const ApplicationIntlContext = createContext();
 
-const contextShape = intlShape;
-
 export default ApplicationIntlContext;
-export { contextShape };
```

Comment:
This is also a breaking change that should be noted in the changelog and upgrade guide. We noted that react-intl doesn't support the intlShape anymore but we should point out that this also means it won't be provided by the ApplicationIntlContext anymore.